api/comments_api.py

```diff
@@ -90,4 +102,19 @@ def do_post(self, feature_id, field_id=None):
     # Callers don't use the JSON response for this API call.
     return {'message': 'Done'}
 
-  # TODO(jrobbins): do_patch to soft-delete comments
+  def do_patch(self, feature_id):
```

Comment:
Nice.  We are real REST programmers now :).

Reply:
>:)